lib/msf/core/db_manager/db_export.rb

```diff
@@ -0,0 +1,20 @@
+require 'msf/core/db_export'
+
+module Msf::DBManager::DbExport
+  def run_db_export(path, format)
+    exporter = Msf::DBManager::Export.new(framework.db.workspace)
+
+    output_file = exporter.send("to_#{format}_file".intern, path) do |mtype, mstatus, mname|
+      if mtype == :status
+        if mstatus == "start"
```

Comment:
Would it be a good idea to make "start" and "complete" class variables available for consistence between this usage and internally within `lib/msf/core/db_export.rb`?